### ISSUE TITLE
Additional columns in Ticket overview

### ISSUE THREAD

**Issue by muppeth** [opensupports/opensupports#424]
The ticket overview is missing some key fields. It would be nice to see state of the ticket (open or closed), who is the ticket assign to, and when was the last reaction/comment on the ticket.

**Comment by hmrneves1** [comment]
Hello you can check this information by hovering #<ticket_number>

**Comment by muppeth** [comment]
@hmrneves1 Oh. Thanks for the hint. However I still think having this info in the general table overview would be more productive as you could see it right away. I think there is plenty of space for that.

**Comment by hmrneves1** [comment]
nop!! I know what you mean, but, in my opinion it would take too much space! Too me, the department isn't really much necessary...

**Comment by nickbe** [comment]
I have to object, yes it definetely is. We're currently testing the system with a dozen departments and lots and lots of tickets. Especially in the "new tickets" and "my tickets" list this would be a lot.

Currently we have real departments and pseudo departments, which we've named after important clients. And important clients in the "new tickets" list are like a red flag for us.

When the new "custom field" feature is coming this might change because from then on we would very much like to use custom fields to fill out a "client name" field. But then this would also mean we have to see it in the list. @ivandiazwm - maybe this could be presented similar to tags. Like maybe a custom field can have a [x] Tag flag which allows us to see its content in lists too.

**Comment by hmrneves1** [comment]
Totally understand the point! Just expressed my point of view ;) to me the app is almost perfect!

**Comment by hmrneves1** [comment]
Personally I don't like the ticket table just because it doesn't show enough information. I have developed a ticket / task system. I uploaded a PScreen to show (to me) how the ticket table should be presented.

https://imgur.com/a/uR0fHkF

**Comment by muppeth** [comment]
@nickbe Indeed the potential is very big. We are considering moving away from zammad (we prefer more projects with no commjunity/enterprise versions). 

@hmrneves1 yeah that screenshot already gives more insight on the situation.

**Comment by mredigonda** [comment]
Yes, we talked about this too in #380 (I'm closing it so we move the discussion here).

We really need a way to make the details of a ticket clear just from looking at the list instead of having to hover over it.
